lsp-tests/Test.hs

```diff
@@ -16,59 +16,65 @@ import Annotation (SRng(DummySRng))
 main :: IO ()
 main = defaultMain $ testGroup "Tests" [hoverTests, hoverTypeInfoTests, typeCheckerTests, unitTests]
 
+-- | Takes 1-indexed range positions and converts them to 0-indexed range positions for LSP-server
+mkRange' :: Int -> Int -> Int -> Int -> Range
+mkRange' l1 c1 l2 c2 = mkRange (l1-1) (c1-1) (l2-1) (c2-1)
```

Comment:
I'd probably generate a SRng and reuse the sRngToRange function instead, but this works fine too. I'll probably make that change later.